Wizardry/Core/BattleSys/Source/BattleHit.c

```diff
@@ -290,6 +290,17 @@ STATIC_DECLAR int BattleHit_CalcDamage(struct BattleUnit * attacker, struct Batt
     }
 #endif
 
+#if (defined(SID_LunarBrace) && (COMMON_SKILL_VALID(SID_LunarBrace)))
+    if (BattleSkillTester(attacker, SID_LunarBrace))
+    {
+        if(&gBattleActor == attacker)
+        {
+            RegisterActorEfxSkill(GetBattleHitRound(gBattleHitIterator), SID_LunarBrace);
```

Comment:
i think no need for anim?

Reply:
I'd like to keep it if possible, because of the file it's in.